test/suites/rt-upgrade-chopsticks-orchestrator/test-upgrade-chain.ts

```diff
@@ -11,37 +11,50 @@ describeSuite({
         let api: ApiPromise;
 
         beforeAll(async () => {
-            api = context.polkadotJs();
+            api = context.pjsApi;
 
-            const rtBefore = api.consts.system.version.specVersion.toNumber();
-            const sessionBefore = api.query.session.currentIndex();
-            log("About to upgrade to runtime at:");
-            log((await MoonwallContext.getContext()).rtUpgradePath);
+            console.dir((await api.call.tanssiUtilApi.sessionPeriod()).toNumber(), { depth: 1 });
```

Comment:
we probably cannot call this before the runtime is up is that right?

Reply:
that's correct

Reply:
can we just do an if runtime > 1100 then call thsi?